Makefile

```diff
@@ -42,6 +42,7 @@ LDLIBS = $(TEST_LIBS)
 SUDO =
 
 CCTZ_LIB = libcctz.a
+CCTZ_STATIC_LIB = $(CCTZ_LIB)
```

Comment:
Yeah, makes space for `CCTZ_SHARED_LIB`. The `CCTZ_LIB` was just left there as backwards compatible variable in case anyone was doing `make CCTZ_LIB=libmycctz.a` for some reason. Happy to remove it!